### PR TITLE
[jsk_topic_tools] add static_tf_republisher.py and documentation

### PR DESCRIPTION
**Pull request by sktometometo**
- add static_tf_republisher.py to republish /tf_static from a rosbag file

### CODE REVIEW

File: doc/jsk_topic_tools/scripts/static_tf_republisher.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+static_transform_republisher.py
+=============================
```
Comment (knorth55) [inline doc/jsk_topic_tools/scripts/static_tf_republisher.rst:2]:
```suggestion
===============================
```

File: doc/jsk_topic_tools/scripts/static_tf_republisher.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+rosrun jsk_topic_tools static_transform_republisher.py _file:=<absolute file path to a rosbag file>
+```
+
+or
+
+```
+rosrun jsk_topic_tools static_transform_republisher.py <file path to a rosbag file>
+```
```
Comment (knorth55) [inline doc/jsk_topic_tools/scripts/static_tf_republisher.rst:23]: can you add document about parameter?
maybe this node only has `~file` parameter.

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: doc/jsk_topic_tools/scripts/static_tf_republisher.rst (hunk 2 above)
Comment (sktometometo) [reply doc/jsk_topic_tools/scripts/static_tf_republisher.rst:23]: Added

File: doc/jsk_topic_tools/scripts/static_tf_republisher.rst (hunk 1 above)
Comment (sktometometo) [reply doc/jsk_topic_tools/scripts/static_tf_republisher.rst:2]: Fixed

**Review by knorth55 — APPROVED** [review]

**Pull request merged**